packages/system/src/consumeStyleProps.ts

```diff
@@ -42,8 +42,6 @@ export type StyledProps<T extends ThemeSystemType = ThemeSystemType> =
     MaskProps &
     ColumnProps<T> &
     BackgroundProps &
-    // eslint-disable-next-line @typescript-eslint/no-duplicate-type-constituents -- FIXME
-    EffectProps &
```

Comment:
EffectProps was duplicate on line41. So remove it.

Reply:
thx :)